test/integration/auth/bootstraptoken_test.go

```diff
@@ -47,23 +46,17 @@ func (b bootstrapSecrets) Get(name string) (*corev1.Secret, error) {
 
 // TestBootstrapTokenAuth tests the bootstrap token auth provider
 func TestBootstrapTokenAuth(t *testing.T) {
-	tokenID, err := bootstraputil.GenerateTokenID()
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	secret, err := bootstraputil.GenerateTokenSecret()
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
```

Comment:
given GenerateTokenID and GenerateTokenSecret are already tested elsewhere.

/sig auth
/lgtm